aten/src/ATen/native/TensorShape.cpp

```diff
@@ -404,12 +441,12 @@ Tensor& set_storage_meta__symint(
   c10::SymDimVector contiguous_strides;
   if (stride.data() == nullptr) {
     // TODO: dedupe this with empty() symbolic logic
-    int64_t dim = size.size();
+    auto dim = size.size();
```

Comment:
I'm not a fan of this kind of auto in the case where the integer width is quite important.